app/MetricUnitConverter.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ConverterApp;
+
+class MetricUnitConverter extends Converter
+{
+public function converter($currentUnit,$newUnit)
```

Comment:
This class allows us to convert values with Metric units. So there should be checking for that in constructor. For that 
We should inherit all metric units from same base class `Metric` which will be inherited from `Unit` class. In constructor you can check if unit of both values passed is of Type `Metric`

`Length` class is not much useful in current scenario.